_installation/mac_os.md

```diff
@@ -0,0 +1,117 @@
+---
+url: mac_os.html
+number: i
+title: Installation on Mac OS X
+layout: rails_tutorial
+---
+
+{:.sectionheader}
+# Mac OS X install
+
+To get a mac ready for Ruby on Rails development, we need to do the following:
+
+1. install command-line tools (XCode)
+1. install ruby
+1. install rails
+
+We're going to do these tasks via the Terminal console, so if you aren't familiar with that, please read **this non-existent page** over first.
+
+## Command-line tools
+
+When you are installing ruby and ruby gems (libraries), your system will often need to compile code for you.  The compilers for this can be installed with XCode, the development tool suite for native mac os development.  If you think you might want to do some mac os x or iOS development, go ahead and install XCode (it's a LARGE download...), but if you just want to ignore this part and get on with Ruby on Rails, you can install just the Command-line tools.
+
+If you aren't sure if you need this, try typing `gcc` in a terminal window. (`gcc` is the name of the compiler and stands for GNU Compiler Collection):
+
+{% highlight bash %}
+   > gcc --version
+{% endhighlight %}
+
+If you get the message, `-bash: gcc: command not found`, install the command-line tools with the command:
+
+{% highlight bash %}
+  > xcode-select --install
+{% endhighlight %}
+
+Note that this will download the command-line tools and install them.  Once installed, `gcc -version` should print out some information about the compiler, e.g.:
+
+{% highlight bash %}
+  > gcc --version
+{% endhighlight %}
+
+This should result in something similar to the following:
+
+{% highlight bash %}
+  Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
+  Apple LLVM version 8.0.0 (clang-800.0.38)
+  Target: x86_64-apple-darwin15.6.0
+  Thread model: posix
+  InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
+{% endhighlight %}
+
+
+## "Easy" ruby installs with rvm
+
+The [Ruby Version Manager (rvm)](http://rvm.io/) will allow you to install different versions of ruby on your machine and switch between them as you switch from one job to another.  This can save you from the pain of upgrading an old project to the newest ruby version when it's running along just fine.
+
+To install rvm, open a terminal window and run the following commands (copied directly off of the rvm website):
+
+{% highlight bash %}
+  > gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+\curl -sSL https://get.rvm.io | bash -s stable
```

Comment:
At first glance, it wasn't clear that this was another thing I needed to run. Maybe all new lines should start with a ">"? I'm not sure - it might've just been difficult to read in the raw markdown file.